### DISCUSSION TITLE
Discussions about whisper-timestamped here and on Discord

### DISCUSSION THREAD

**Discussion by Jeronymous** [linto-ai/whisper-timestamped#43] [Announcements]
## 👋 Welcome!

We’re using this github Discussions forum as a place to ask questions, share ideas, and suggest improvements or new features for `whisper-timestamped`.

You can also join the Discord thread on LinTO Community https://discord.gg/aSVvNuzc9u

**Comment by jon-fox** [comment]
This discord link seems to have expired

**Reply by Jeronymous** [reply]
Thanks a lot for notifying. This is fixed

**Reply by jon-fox** [reply]
thanks a lot for updating
